src/sentry/api/endpoints/issue_occurrence.py

```diff
@@ -41,8 +44,28 @@ def post(self, request: Request) -> Response:
         Write issue occurrence and event data to a Kafka topic
         ``````````````````````````````````````````````````````
         :auth: superuser required
+        :pparam: string dummy: pass 'True' to load a dummy event instead of providing one in the request
         """
-        event = request.data.pop("event")
+        event = {}
+        if request.query_params.get("dummy") == "True":
+            event = {
+                "event_id": "44f1419e73884cd2b45c79918f4b6dc4",
+                "project_id": 1,
```

Comment:
we might want to grab a valid `project_id` here. is there any way we can tie a call to an org? I'm pretty out of the loop on the level of validation we're doing here

Reply:
Mmmm might be able to grab the user from the request and find an org they belong to?

Reply:
ah got it. then yeah if we can I think. this is a temporary feature to help expedite e2e testing yeah? so whatever makes the most sense for that use case